packages/realm-app-importer/src/AppConfigBuilder.ts

```diff
@@ -37,7 +37,7 @@ export type SyncConfig = {
 export type ServiceConfig = {
   name: string;
   type: string;
-  config: DisabledSyncConfig | PartitionSyncConfig | FlexibleSyncConfig;
```

Comment:
Instead of deleting this, I think https://github.com/realm/realm-js/blob/main/packages/realm-app-importer/src/AppConfigBuilder.ts#L126 should be renamed to `DisabledSyncConfig`.

Reply:
Sounds good to me.